lib/ddtrace/transport.rb

```diff
@@ -16,6 +17,7 @@ class HTTPTransport
 
     # header containing the number of traces in a payload
     TRACE_COUNT_HEADER = 'X-Datadog-Trace-Count'.freeze
+    RUBY_INTERPRETER = RUBY_VERSION > '1.9' ? RUBY_ENGINE + '-' + RUBY_PLATFORM : 'ruby-' + RUBY_PLATFORM
```

Comment:
Nitpicking, but could you please add a test for this (this is typically the kind of not-so-easy-to-spot feature, when it goes wrong). There's already a test checking that the trace count header is sent: https://github.com/DataDog/dd-trace-rb/blob/master/test/tracecountheader_test.rb#L22 you may create a similar test reading the values of your new headers.